security/response-process.md

```diff
@@ -0,0 +1,254 @@
+Michael responds to the report, cc'ing security@openquantumsafe.org, and proceeds directly to the Patching phase.
+
+### Reports from other channels
+
+Reporters may not always follow instructions, and it is possible that security reports are received through other, non-endorsed channels (for example, email sent directly to a member of the VMT or a direct message on the PQCA discord).
+Such a report should be funnelled into one of the provided channels as the member of the VMT who first becomes aware of it sees fit.
+The focus should be on addressing the vulnerability report, not forcing the reporter(s) to resubmit via another channel.
+
+## 2. Assessment
+
+This phase begins when a vulnerability report is acknowledged by the VMT.
+It ends when the VMT responds to the reporter(s) with an assessment of the report.
+
+### Appointing a domain expert
+
+The Quarterback identifies the person or people who are most qualified to assess the report and, if necessary, develop a fix.
+The Quarterback provides them with all necessary information.
+These domain experts may not be members of the VMT, but they should be people whom the VMT trusts to handle security-related information.
+It is possible that the Quarterback will also be the team member who is most qualified to assess the issue.
+
+### Assessing the report
+
+The domain expert(s) assess the report and determine what action, if any, is required from OQS.
+If more information from the reporter(s) is needed, the Quarterback facilitates communication.
+The below table, indicating possible assessment outcomes, is copied from the [OSSF's vulnerability guide](https://github.com/ossf/oss-vulnerability-guide/blob/da8a22a1b09636c20a93fed58c1ba179557358d0/maintainer-guide.md), with the exception of the "Out of scope" lines, which are unique to OQS.
+
+| Assessment          | Response                                                                                                                                                                                                                                                                                                                                                               |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Working as intended | Let the reporter know this is the intended behavior. If they think this behavior could be improved, they can file a feature request. Close the security issue. When responding with this assessment, you should explain why you arrived at this conclusion, in case the original report was unclear and the VMT has unintentionally misunderstood the original report. |
```

Comment:
Confusing use of "you". Suggest to re-phrase "VMT responder should...."

Reply:
Or would it be "Quarterback"? Can't say I'm the biggest fan of using the QB terminology, but nevertheless whatever term is used should not be interchanged with other terms that represent the same role in the process.